code/game/objects/machinery/hologram.dm

```diff
@@ -5,7 +5,7 @@
 	name = "holopad"
 	desc = "It's a floor-mounted device for projecting holographic images."
 	icon_state = "holopad0"
-	layer = LOW_OBJ_LAYER
+	layer = XENO_WEEDS_LAYER
```

Comment:
Are you sure you dont want to make an inbetween so weeds can never appear below a holopad?

Reply:
I don't know. But this comes off somewhat as a suggestion, so i'll do that.